integration-libs/cdc/user-profile/update-profile/cdc-update-profile.module.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * SPDX-FileCopyrightText: 2022 SAP Spartacus team <spartacus-team@sap.com>
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { RouterModule } from '@angular/router';
+import { NgSelectModule } from '@ng-select/ng-select';
+import {
+  AuthGuard,
+  CmsConfig,
+  I18nModule,
+  provideDefaultConfig,
+  UrlModule,
+} from '@spartacus/core';
+import {
+  FormErrorsModule,
+  NgSelectA11yModule,
+  SpinnerModule,
+} from '@spartacus/storefront';
+import {
+  UpdateProfileComponent,
+  UpdateProfileComponentService,
+} from '@spartacus/user/profile/components';
+import { CDCUpdateProfileComponentService } from './cdc-update-profile-component.service';
+
+@NgModule({
+  imports: [
+    CommonModule,
+    FormsModule,
+    ReactiveFormsModule,
+    SpinnerModule,
+    I18nModule,
+    FormErrorsModule,
+    RouterModule,
+    UrlModule,
+    NgSelectModule,
+    NgSelectA11yModule,
+  ],
+  providers: [
+    provideDefaultConfig(<CmsConfig>{
+      cmsComponents: {
+        UpdateProfileComponent: {
+          component: UpdateProfileComponent,
+          guards: [AuthGuard],
```

Comment:
Is this _really_ needed? 
I believe we'd like to only overwrite the service. And guard and component reference are already defined in the original UpdateProfileModule 
https://github.com/SAP/spartacus/blob/f3518af089b5bba39563a5c6778310ad931ba4bf/feature-libs/user/profile/components/update-profile/update-profile.module.ts#L48-L49

Reply:
noted, thanks